doc/md/guides/orms/gorm.md

```diff
@@ -70,14 +70,14 @@ go get -u ariga.io/atlas-provider-gorm
 ### Standalone vs Go Program mode
 
 The Atlas GORM Provider can be used in two modes:
-* **Standalone** - If all of your GORM models exist in a single package, and either embed `gorm.Model` or contain `gorm` struct tags, 
+* **Standalone** - If all of your GORM models and views exist in a single package, and the models either embed `gorm.Model` or contain `gorm` struct tags, 
   you can use the provider directly to load your GORM schema into Atlas.
-* **Go Program** - If your GORM models are spread across multiple packages, or do not embed `gorm.Model` or contain `gorm` struct tags, 
+* **Go Program** - If your GORM models or views are spread across multiple packages, or the models do not embed `gorm.Model` or contain `gorm` struct tags, 
   you can use the provider as a library in your Go program to load your GORM schema into Atlas.
 
 ### Standalone mode
 
-If all of your GORM models exist in a single package, and either embed `gorm.Model` or contain `gorm` struct tags,
+If all of your GORM models and [view definition](https://github.com/ariga/atlas-provider-gorm?tab=readme-ov-file#views) exist in a single package, and the models either embed `gorm.Model` or contain `gorm` struct tags,
```

Comment:
No need to mention it in every sentence. Models can be presented as views as well.

Reply:
Agree. Leave it on L55 and in the rest refers to it as "models"